packages/dapp/src/types/userTracking.types.ts

```diff
@@ -39,11 +39,16 @@ export interface trackPageloadProps {
 }
 
 export interface TrackConnectWalletProps {
-  account: WalletAccount;
+  account?: WalletAccount;
   data?: { [key: string]: string | number | boolean };
   disableTrackingTool?: EventTrackingTool[];
   disconnect?: boolean;
 }
+
+export interface TrackDisconnectConnectWalletProps {
```

Comment:
Should it be `TrackDisconnectWalletProps`? 👀

Reply:
Right! Thanks

Reply:
I renamed them in last commit